src/main/java/uk/gov/hmcts/reform/coh/domain/OnlineHearingState.java

```diff
@@ -14,9 +17,11 @@
 
     @Id
     @Column(name = "online_hearing_state_id")
+    @JsonIgnore
     private int onlineHearingStateId;
 
     @Column(name = "state")
+    @JsonProperty("state_name")
```

Comment:
Remove this JSON annotations because there is no code to render the online hearing state in the response yet.